### PR TITLE
PIM-4920: add translation keys

### PR DESCRIPTION
**Pull request by skeleton**

### CODE REVIEW

**Comment by jmleroux** [comment]
GTM

File: src/Pim/Bundle/UIBundle/Resources/translations/messages.en.yml [hunk 1]
```diff
@@ -35,3 +35,5 @@ Most Viewed: Most Viewed
 "Add this page to favorites": "Add this page to favorites"
 "Minimize the window to the pinbar": "Minimize the window to the pinbar"
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/translations/messages.en.yml]: :cactus:

File: src/Pim/Bundle/UIBundle/Resources/translations/messages.en.yml [hunk 2]
```diff
@@ -35,3 +35,5 @@ Most Viewed: Most Viewed
...
 "Loading ...": "Loading ..."
+"Yes": "Yes"
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/translations/messages.en.yml]: :racehorse:

**Comment by jjanvier** [comment]
not GTM n00b

we don't translate in other languages, we have to use Crowdin

File: src/Pim/Bundle/UIBundle/Resources/translations/messages.fr.yml [hunk 3]
```diff
@@ -35,3 +35,5 @@ Most Viewed: Plus populaires
 Add this page to favorites: Ajouter cette page à vos favoris
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/translations/messages.fr.yml]: :camel:

File: src/Pim/Bundle/UIBundle/Resources/translations/messages.en.yml [hunk 4]
```diff
@@ -35,3 +35,5 @@ Most Viewed: Most Viewed
 "Add this page to favorites": "Add this page to favorites"
 "Minimize the window to the pinbar": "Minimize the window to the pinbar"
 "Loading ...": "Loading ..."
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/translations/messages.en.yml]: :dancers:

File: src/Pim/Bundle/UIBundle/Resources/translations/messages.en.yml [hunk 5]
```diff
@@ -35,3 +35,5 @@ Most Viewed: Most Viewed
 "Add this page to favorites": "Add this page to favorites"
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/translations/messages.en.yml]: :m

File: src/Pim/Bundle/UIBundle/Resources/translations/messages.fr.yml [hunk 6]
```diff
@@ -35,3 +35,5 @@ Most Viewed: Plus populaires
...
 Minimize the window to the pinbar: Réduire la fenêtre dans la pinbar
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/translations/messages.fr.yml]: :yen:

File: src/Pim/Bundle/UIBundle/Resources/translations/messages.fr.yml [hunk 7]
```diff
@@ -35,3 +35,5 @@ Most Viewed: Plus populaires
...
 Loading ...: Chargement ...
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/translations/messages.fr.yml]: :kimono:

File: src/Pim/Bundle/UIBundle/Resources/translations/messages.fr.yml [hunk 8]
```diff
@@ -35,3 +35,5 @@ Most Viewed: Plus populaires
...
+"Yes": "Oui"
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/translations/messages.fr.yml]: :v:

**Comment by solivier** [comment]
:-1:

**Comment by jjanvier** [comment]
:-1:

File: src/Pim/Bundle/UIBundle/Resources/translations/messages.fr.yml (hunk 8 above)
Comment (jjanvier) [inline src/Pim/Bundle/UIBundle/Resources/translations/messages.fr.yml]: :-1:

**Comment by jmleroux** [comment]
GTM

**Comment by jjanvier** [comment]
GTM

**Pull request merged**